omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -110,11 +114,24 @@ public String get(PageModel model,
 			Context.removeProxyPrivilege(GET_LOCATIONS);
 		}
 
+		model.addAttribute("showSessionLocations", !isLocationUserPropertyAvailable(administrationService));
+		Object showLocation = pageRequest.getAttribute("showSessionLocations");
```

Comment:
I still do not understand why you call model.addAttribute twice instead of once

Reply:
Can't you do it once?

Reply:
Done @dkayiwa